.github/workflows/cuda/Linux-env.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 
 case ${1} in
+  cu128)
+    export FORCE_CUDA=1
+    export PATH=/usr/local/cuda-12.8/bin:${PATH}
+    export TORCH_CUDA_ARCH_LIST="7.5;8.0;8.6;9.0;10.0;12.0+PTX"
+    ;;
```

Comment:
The list of architectures here is consistent with PyTorch 2.7 built with CUDA 12.8.